setup.py

```diff
@@ -13,6 +23,12 @@
     ],
     include_package_data=True,
     license='GPLv3',
+    packages=find_packages(exclude=["test*.py"]),
     package_data={'': ['LICENSE']},
     url='https://github.com/quipucords/quipucords',
+    # non-python scripts go here
```

Comment:
This says non-python scripts go here, but `bin/qpc` is Python. No request for change, I just don't understand what this does.

Reply:
fixed. copy & paste from rho. I don't understand it either.